### ISSUE TITLE
Feature request: Downloadable packages.

### ISSUE THREAD

**Issue by kalwalt** [Carnaux/NFT-Marker-Creator#38]
It will be nice if it will be possible to download the project and the apps as a zip file or a npm package. This would be the preferable action for user that are not used to git and all git commands/procedures.
I'm writing a little tutorial and i'm facing this "problem" that if you want to use the node app you are obliged to install git and run git clone to download the project.

**Comment by Carnaux** [comment]
You actually can use the "Clone or Download" button in the project page, it already comes in a zip file.

**Comment by kalwalt** [comment]
> You actually can use the "Clone or Download" button in the project page, it already comes in a zip file.

@Carnaux yes you are right! i forgot this... but making a tag and a release will let you add also specific package to download, for example:

- a package with only the node app and the required libs.
- a package with only the web app and the required  libs.

so with a wget command you can download the desired package. 
I will change my article adding the download option,  i think is much simpler.

**Comment by andreyrd** [comment]
Publishing this as an npm module would be amazing
